bin/index.js

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env node
+
+import { program } from 'commander';
+import fs from 'fs-extra';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { execSync } from 'child_process';
+import figlet from 'figlet';
+import chalk from 'chalk';
+import { createSpinner } from 'nanospinner';
+import { templates } from './configs.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const template = templates.basic;
+
+program
+  .version('v1.0.0','-v, --version')
+  .description('A simple Express server generator');
+
+program
+  .command('init')
+  .description('Initialize a new Express server')
+  .action(() => {
+    
+    toolIntro();
+
+    console.log('Starting server initialization...');
+    
+    const targetDir = process.cwd();
+    const parentDir = path.dirname(__dirname)
+    const templatePath = path.join(parentDir, 'templates',template.name);
```

Comment:
nit: leave 'space' after ',' to separate parameters.
```suggestion
    const templatePath = path.join(parentDir, 'templates', template.name);
```

Reply:
alright